{
  "modified": "2024-06-25T13:37:06Z",
  "published": "2024-06-25T13:37:06Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5346",
  "summary": "Malicious code in mallodys (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "mallodys",
        "purl": "pkg:pypi/mallodys"
      },
      "versions": [
        "1.0"
      ]
    }
  ],
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-04128",
        "import_time": "2024-06-28T02:49:24.036173229Z",
        "modified_time": "2024-06-25T13:37:06Z",
        "sha256": "15aa961809993e57e6ba42e560e89be465ded37be8302df5f9fdbdacdb8a7a69",
        "source": "reversing-labs",
        "versions": [
          "1.0"
        ]
      }
    ]
  }
}
